package.json

```diff
@@ -7,11 +7,6 @@
     "doc": "doc"
   },
   "scripts": {
-    "ng": "ng",
-    "start": "ng serve",
-    "build": "ng build",
-    "lint": "eslint src/iotMapManager/**/*.ts",
```

Comment:
I probably should move this entire `eslint` configuration to `src/iotMapManager`. Let's try it in `master` branch before to see if it was working.